libs/sdk-ui-kit/src/Header/HeaderHelp.tsx

```diff
@@ -128,7 +133,7 @@ export const CoreHeaderHelp: React.FC<IHeaderHelpProps> = ({
             <FormattedMessage id="gs.header.help" />
         </a>
     ) : (
-        <div className={classNames} onClick={() => toggleHelpMenu()}>
+        <div className={classNames} onClick={() => toggleHelpMenu()} ref={helpMenuRef}>
```

Comment:
Can we simply pass `toggleHelpMenu` here instead, something like:

```
<div className={classNames} onClick={() => toggleHelpMenu()} ref={helpMenuRef}>
```

↓

```
<div className={classNames} onClick={toggleHelpMenu} ref={helpMenuRef}>
```

Reply:
<img width="983" alt="Screen Shot 2022-07-26 at 15 00 54" src="https://user-images.githubusercontent.com/5329091/180955071-989a9397-1e8c-4037-b6af-2b9dfde2985c.png">
We can't, it gets error tslint.

Reply:
ok then